vignettes/views.Rmd

```diff
@@ -89,11 +89,22 @@ data["class"] <- c("V", "VI")
 synStore(Table(view$properties$id, data))
 ```
 
-The change in annotations reflect in synGetAnnotations():
+The change in annotations is reflected in synGetAnnotations():
 ```{r collapse=TRUE, eval=FALSE}
 synGetAnnotations(file2$properties$id)
 ```
 
+A unique etag is associated with every file that updates when changes are made to the file annotations or metadata. Any updates pushed to Synapse will change an object's etag.
+```{r collapse=TRUE, eval=FALSE}
+data$etag
+```
+
+There may be cases where you want to update the annotations on a subset of files in a view. In order to preserve the etag, and thus the file history, you will need to store only the rows that have been modified. 
```

Comment:
does the version of the file change (what I would call the file history) or only the etag?

This also generalizes to other types of views (project views, or views with folders in them too) - and these are not 'versionable' entities, so their 'history' is changing via the etag. Confusing!

Reply:
The etag changes and the "modifiedOn" changes (a component of the file history, I think) while the file version remains the same.

However, I don't think this is the appropriate place in the documentation to address those nuances.